pkg/hostagent/agent.go

```diff
@@ -394,3 +394,14 @@ func (agent *HostAgent) Run(stopCh <-chan struct{}) {
 
 	agent.cleanupSetup()
 }
+func deepCopyObj(src, dst interface{}) error {
```

Comment:
would it make sense to put this in util package?

Reply:
moved to util